airbyte-webapp/src/hooks/services/Notification/types.ts

```diff
@@ -1,13 +1,6 @@
-import React from "react";
+import { ToastProps } from "components/ui/Toast";
 
-export interface Notification {
-  id: string | number;
-  title: React.ReactNode;
-  text?: React.ReactNode;
-  isError?: boolean;
-  nonClosable?: boolean;
-  onClose?: () => void;
-}
+export type Notification = ToastProps & { id: string | number; nonClosable?: boolean };
```

Comment:
`export interface Notification extends ToastProps {` will give us better reference checking in VScode

Reply:
Fixed